src/routes/RequestsRoute.js

```diff
@@ -181,7 +189,37 @@ export const urls = {
   },
 };
 
-export const getListFormatter = (getRowURL, setURL) => ({
+export const getListFormatter = (
+  {
+    getRowURL,
+    setURL
+  },
+  {
+    selectedRows,
+    pickSlipsToCheck,
+    pickSlipsData,
+    getPrintContentRef,
+    pickSlipsPrintTemplate,
+    toggleRowSelection,
+    onBeforeGetContentForSinglePrintButton
+  }
+) => ({
+  'select': ({
+    // eslint-disable-next-line react/prop-types
```

Comment:
could you remove this comment and add property to propTypes?
In this case we will not have eslint error.

Reply:
I may need some help with this. finding it tricky to add propTypes for this one.

Reply:
Could you please clarify can we write code in other way? Also it will help to fix eslint error. Just for example:
`  'select': rq => (
    <CheckboxColumn
      rq={rq}
      selectedRows={selectedRows}
      toggleRowSelection={toggleRowSelection}
    />
  ),`
`CheckboxColumn.js`
`import React from 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';

import { Checkbox } from '@folio/stripes/components';

const CheckboxColumn = ({
  rq,
  selectedRows,
  toggleRowSelection,
}) => {
  const {
    id,
    ...rowData
  } = rq;

  return (
    <div // eslint-disable-line jsx-a11y/click-events-have-key-events
      tabIndex="0"
      role="button"
      onClick={e => e.stopPropagation()}
    >
      <Checkbox
        checked={Boolean(selectedRows[id])}
        aria-label={<FormattedMessage id="ui-inventory.instances.rows.select" />}
        onChange={() => toggleRowSelection({
          id,
          ...rowData
        })}
      />
    </div>
  );
};

CheckboxColumn.propTypes = {
  rq: PropTypes.shape({
    id: PropTypes.string.isRequired
  }),
  selectedRows: PropTypes.func.isRequired,
  toggleRowSelection: PropTypes.func.isRequired,
};

export default CheckboxColumn;
`